core/src/main/resources/error/error-classes.json

```diff
@@ -105,6 +105,10 @@
     "message" : [ "The fraction of sec must be zero. Valid range is [0, 60]. If necessary set <config> to false to bypass this error. " ],
     "sqlState" : "22023"
   },
+  "INVALID_FUNCTION_ARGUMENTS" : {
+    "message" : [ "Invalid <invalidType> of arguments for function <functionName>. <message>" ],
```

Comment:
Could you use sub-classes here, see an example in https://github.com/apache/spark/pull/36307

Reply:
@MaxGekk Thanks. I will be use sub-classes to correct it later.